### PR TITLE
add sum from 1 to n in python and python.md

### PR DESCRIPTION
**Pull request by hemt1912**
- delete NEWFILE
- add python.md
- add file to sum from 1 to n

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+**/__pycache__
```
Comment (sonhan) [inline .gitignore]: add new line to the end of file plz

File: sum.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+def sum_1_to_n(n):
+    answer = 0
+    for i in range(1, n+1):
+        answer += i
+    return(answer)
```
Comment (sonhan) [inline sum.py]: add new line to the end of file plz

File: python.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# How to test function in terminal
+- python -c "from (file name) import(function); code_to_use function"
```
Comment (sonhan) [inline python.md]: how to define a function in python

**Comment by hemt1912** [comment]
Done!

File: .gitignore (hunk 1 above)
Comment (hemt1912) [reply .gitignore]: I did it

File: sum.py (hunk 2 above)
Comment (hemt1912) [reply sum.py]: finish

File: python.md (hunk 3 above)
Comment (hemt1912) [reply python.md]: Done

**Review by sonhan — APPROVED** [review]

**Pull request merged**